dev/set_matrix.py

```diff
@@ -464,8 +472,8 @@ def main():
     flavors = set(x.split("-")[0] for x in config.keys())
     changed_flavors = get_changed_flavors(changed_files, flavors)
 
-    matrix = set(expand_config(config))
-    matrix_ref = set(expand_config(config_ref))
+    matrix = set(expand_config(config, args.exclude_dev_versions))
+    matrix_ref = set(expand_config(config_ref, args.exclude_dev_versions))
```

Comment:
![image](https://user-images.githubusercontent.com/17039389/122863813-75b33f00-d35e-11eb-825d-77c8e3a89ce9.png)

Confirmed the dev versions have been removed.